docs/topics/api/reviewers.rst

```diff
@@ -131,7 +131,7 @@ This endpoint allows you to browse through the contents of an Add-on version.
     :>json string file.content: Raw content of the requested file.
     :>json string file.selected_file: The selected file, either from the ``file`` parameter or the default (manifest.json, install.rdf or package.json for Add-ons as well as the XML file for search engines).
     :>json array file.entries[]: The complete file-tree of the extracted XPI.
-    :>json boolean|string file.entries[].binary: ``True`` if the file is a binary file (e.g an .exe, dll, java, swf file), ``'image'`` if the file is an image or ``False`` otherwise. If ``False`` or ``'image'`` the file should be presentable to the user.
+    :>json boolean file.entries[].binary: ``True`` if the file is a binary file (e.g an .exe, dll, java, swf file) or ``False`` otherwise. If ``False`` the file should be presentable to the user.
```

Comment:
Didn't notice that before: should we say `class` and not `java` ? `java` files are text, technically :)

Reply:
eh, indeed

Reply:
This needs updating anyway

Reply:
Updated in 2c0d38b690